addons/product/models/product.py

```diff
@@ -351,6 +351,9 @@ def search(self, args, offset=0, limit=None, order=None, count=False):
             args.append((('categ_id', 'child_of', self._context['search_default_categ_id'])))
         return super(ProductProduct, self).search(args, offset=offset, limit=limit, order=order, count=count)
 
+    def _prepare_variant_name(self, attribute_value_ids):
+        return ", ".join([v.name for v in attribute_value_ids])
+
```

Comment:
wouldn't it not be better to have this method as an @api.multi of the product.attribute.value class?
Also, may be _prepare_* is confusing because usually it's used to prepare a value to be written within a create or write. Here I think it's mostly aimed at providing a value for reading. So may be just 
```
@api.multi
_variant_name(self):
```
in the product.attribute.value class would be enough. Thoughts?